### DISCUSSION TITLE
5.1.0 release?

### DISCUSSION THREAD

**Discussion by lukalt** [dmulloy2/ProtocolLib#2476] [General]
MC 1.20.1 has been released a while ago and protocol lib offers no stable release for this so far.
The latest dev build supports 1.20 without any mayor issues so far.
Is there anything still blocking the 5.1.0 release with official 1.20.1 support? @dmulloy2

**Comment by shimajima** [comment]
Hello everyone, we are waiting for release of 1.20.X compatible version as well. Just wanted to chime in.

**Reply by lukalt** [reply]
You can download a working development build for 1.20.1 here: https://ci.dmulloy2.net/job/ProtocolLib/661/artifact/build/libs/ProtocolLib.jar
